client/asset/eth/eth.go

```diff
@@ -843,53 +918,70 @@ func recoverPubkey(msgHash, sig []byte) ([]byte, error) {
 	return pubKey.SerializeUncompressed(), nil
 }
 
-// ReserveN locks funds for redemption. It is an error if there is insufficient
-// spendable balance. Part of the AccountRedeemer interface.
-func (eth *ExchangeWallet) ReserveN(n, feeRate uint64, assetVer uint32) (uint64, error) {
+// ReserveNRedemption locks funds for redemption. It is an error if there
+// is insufficient spendable balance. Part of the AccountLocker interface.
+func (eth *ExchangeWallet) ReserveNRedemption(n, feeRate uint64, assetVer uint32) (uint64, error) {
```

Comment:
Never mind my previous comment about FeeRate for refund or redeem txns.  On matrix we agreed it's safest and simplest to use the MaxFeeRate as the max gas price for the (dynamic) refund and redeem txns because nothing is gained by using a lower estimate.